src/skvhttpclient/src/skvhttp/dto/SKVRecord.h

```diff
@@ -237,6 +237,35 @@ class SKVRecordBuilder {
         _record.storage.serializedCursor++;
         ++_record.fieldCursor;
     }
+    
```

Comment:
The rvalue& let's us pass temporaries directly, like concatenating two strings in place or something like that

Reply:
const& can do that too. You can pass temporary to method taking const&

Reply:
Is that new? So we should use const& everywhere then? In the k23si client we have some interfaces pass by value and I thought that was because it was the most compatible. Anyway, I can remove the rvalue& implementation from this.

Reply:
const& would prevent u from passing/receiving ownership. In this case, we don't need to pass ownership since the receiver doesn't need the object past the call - the msgpack implementation cannot move data into the stream for now. In the k23si client, we want ownership since we can move data instead of copying it.